Sources/Datadog/Logging/LoggingV2Configuration.swift

```diff
@@ -27,6 +27,17 @@ internal func createLoggingConfiguration(
     )
 }
 
+internal func mapInternalAttributeKey(_ originalAttribute: String) -> String {
+    switch originalAttribute {
+    case "application.id":
+        return "application_id"
+    case "session.id":
+        return "session_id"
+    default:
+        return originalAttribute
+    }
+}
```

Comment:
suggestion/ Could we declare constants for `"application.id"` and `"session.id"` as part of `RUMDependency` namespace defined in `Logging` module? Similar to how we define [`RUMDependency`](https://github.com/DataDog/dd-sdk-ios/blob/ncreated/RUMM-3320/GH-1292-fix-apm-with-rum/DatadogSessionReplay/Sources/DatadogCoreIntegration/RUMDependency.swift) in Session Replay - single place with listing all attributes that features exchange between each other.

cc @maxep

Reply:
Because we work with V1 code (no modules yet), it would have to be `enum LogsRUMDependency` (or similar) to scope it properly.

Reply:
Having constants makes sense, should be defined alongside the message receiver IMO.

Reply:
Message receiver can be a good place to define "received" attributes. Where to define "provided" attributes?

Looking at mentioned [RUMDependency](https://github.com/DataDog/dd-sdk-ios/blob/ncreated/RUMM-3320/GH-1292-fix-apm-with-rum/DatadogSessionReplay/Sources/DatadogCoreIntegration/RUMDependency.swift) in SR there are two parts:
```swift
internal enum RUMDependency {
   // MARK: Contract from RUM to SR:
   static let rumBaggageKey = "rum"
   static let serverTimeOffsetKey = "server_time_offset"
   static let ids = "ids"

   // MARK: Contract from SR to RUM (mirror of `SessionReplayDependency` in RUM):
   static let srBaggageKey = "session-replay"
   static let hasReplay = "has_replay"
}
```
and I believe it is good and clear enough - single place describing the whole contract.

If we want to separate it, it's OK for me as long as it is clear and consistent across all modules.

Reply:
> and I believe it is good and clear enough - single place describing the whole contract.

Yep, totally. Just to add my 2 cents: I don't think we should enforce any convention across modules on that. Some Features will use/share a lot of context values, some won't. We can have some flexibility on how Features are implemented internally IMHO